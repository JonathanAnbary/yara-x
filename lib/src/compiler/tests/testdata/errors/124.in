rule test {
  condition:
  	9223372036854775807 + 1000000000 == 0
}