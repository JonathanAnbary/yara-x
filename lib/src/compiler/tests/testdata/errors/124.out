error: number out of range
 --> line:3:4
  |
3 |    9223372036854775807 + 1000000000 == 0
  |    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ this number is out of the allowed range [-9223372036854775808-9223372036854775807]
  |